iPetCare/Application/Services/ExaminationParameterValueService.cs

```diff
@@ -172,7 +172,7 @@ public async Task<ServiceResponse<UpdateExaminationParameterValueDtoResponse>> U
             return new ServiceResponse<UpdateExaminationParameterValueDtoResponse>(HttpStatusCode.BadRequest, "Nie nastąpiło zapisanie do bazy danych");
         }
 
-        private async Task<bool> CanEditExaminationParameterValueAsync(Pet pet)
+        private bool CanEditExaminationParameterValueAsync(Pet pet)
```

Comment:
Skąd zmiana z asynchronicznej wersji na synchroniczną? 
Jeśli jednak jest uzasadniona, to słowo Async powinno również zniknąć z nazwy metody :)